Muszilla/src/model/DbConnect.php

```diff
@@ -0,0 +1,60 @@
+<?php 
+
+namespace App\model;
+
+use App\Controller\HomeController;
+use mysqli;
+
+/**
+ * Class to handle the connection to the database.
+ * 
+ * @var $servername
+ *    Stores the working server name.
+ * 
+ * @var $username 
+ *    Stores the username of the user working on the database.
+ * 
+ * @var $password 
+ *    Stores the password of the user.
+ * 
+ * @var $dbname
+ *    Stores the name of the database to be used. 
+ */ 
+class DbConnect {
+  private $servername = '';
```

Comment:
Add comment about these variables.

Reply:
Already added in class description, Kindly check.